### PR TITLE
SRV-1578: Stripping whitespaces from PRs names

### PR DESCRIPTION
**Pull request by asychev**
https://onetwotripdev.atlassian.net/browse/SRV-1578

Checking for wrong PRs names:
[OTT-9944 release 24.08.2016 12trip] - OK
[FE-1100-f-old-typo-error] - OK
[FE-1099-f-old-amex-bug] - OK
[FE-1135-revert-merge] - OK
[FE-899-vas-text-test--for-merge] - OK
[OTT-9944 release 24.08.2016 12trip xjsx] - OK
[OTT-9567 cr nan value] - OK
[OTT-9944 release 24.08.2016 avia] - OK
[OTT-9944 release 24.08.2016 bo] - OK
[ OTT-9944 release 24.08.2016 tlive] - WRONG! Stripped. Bad guy: Alexey Sutiagin
[OTT-9944 release 24.08.2016 seo] - OK
Selected PRs:
OTT-9944 release 24.08.2016 12trip
OTT-9944 release 24.08.2016 12trip xjsx
OTT-9944 release 24.08.2016 avia
OTT-9944 release 24.08.2016 bo
OTT-9944 release 24.08.2016 tlive
OTT-9944 release 24.08.2016 seo

### CODE REVIEW

File: lib/scenarios/deployrelease.rb [hunk 1]
```diff
@@ -15,6 +15,18 @@ def run
       labels = labels.uniq
 
       prs = issue.related['pullRequests']
+
+      puts 'Checking for wrong PRs names:'
+
+      prs.each do |pr|
+        prname = pr['name'].dup
+        if pr['name'].strip!.nil?
```
Comment (dshmelev) [inline lib/scenarios/deployrelease.rb]: Delete nil? and ship it

**Comment by dshmelev** [comment]
:+1:

**Comment by vkarnushin** [comment]
:+1:

File: lib/scenarios/deployrelease.rb (hunk 1 above)
Comment (asychev) [inline lib/scenarios/deployrelease.rb]: Оставил, нужна проверка именно на nil, ИМХО, так читаемо и очевидно.

**Pull request merged**